server/pkg/item/builder.go

```diff
@@ -74,6 +77,11 @@ func (b *Builder) Project(pid ProjectID) *Builder {
 	return b
 }
 
+func (b *Builder) Thread(tid ThreadID) *Builder {
+	b.i.thread = tid
+	return b
+}
+
```

Comment:
```suggestion
func (b *Builder) Thread(tid ThreadID) *Builder {
	b.i.thread = tid
	return b
}

func (b *Builder) NewThread() *Builder {
	b.i.thread = NewThreadID()
	return b
}

```

NewThread method may be helpful